### DISCUSSION TITLE
Update Git submodules?

### DISCUSSION THREAD

**Discussion by JrobT** [renovatebot/renovate#16411] [Request Help] [answered]
Is there a way to get Renovate to check for any updates to Git submodules when raising a PR? I've had a few failures recently where the fix has been for me to go and update the submodule on the PR manually. If there were some way of getting Renovate to commit this for me, it would save me a job. Thanks

**Comment by electriquo** [comment, marked as answer]
Enable the [Git Submodules](https://docs.renovatebot.com/modules/manager/git-submodules/) package manager

```json
{
  "git-submodules": {
    "enabled": true
  }
}
```
